WSL/install-win10.md

```diff
@@ -113,6 +113,47 @@ wsl --set-default-version 2
 
 This will set the version of any new distribution installed to WSL 2.
 
+## Install and upgrade all distros 
+
+Run the following command in PowerShell for upgrade all your distros and install wsl update package.
```

Comment:
I guess better wording will be:
- run (...) **to** ungrade
or
- run (...) for initiating and upgrade of all...

Otherwise does not sound right.